x-pack/plugins/security/server/authentication/index.test.ts

```diff
@@ -336,4 +337,31 @@ describe('setupAuthentication()', () => {
       );
     });
   });
+
+  describe('createAPIKey()', () => {
+    let createAPIKey: jest.Mock;
```

Comment:
__nit:__ I believe it's not a `jest.Mock`, but ` let createAPIKey: (
      request: KibanaRequest,
      body: CreateAPIKeyOptions['body']
    ) => Promise<CreateAPIKeyResult | null>;`?